projects/github.com/realm/SwiftLint/package.yml

```diff
@@ -0,0 +1,25 @@
+distributable:
+  url: https://github.com/realm/SwiftLint/archive/refs/tags/{{version}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: realm/SwiftLint/releases/tags
+
+platforms:
+  - darwin
+
+provides:
+  - bin/swiftlint
+
+warnings:
+  - vendored
+
+build:
```

Comment:
probs: `working-dir: build` would be the easiest way around these issues.

At some point I'll make using vendor compiled binaries more official. Though I would like the binaries to be GPG signed and few people do that.